src/lib/util.js

```diff
@@ -297,6 +297,11 @@ async function processSpecification(spec, callback, args, counter) {
             path: path.resolve(__dirname, '../../builds/browser.js')
         });
 
+        // Import WebIDL2, needed to parse definitions out of the HTML spec
+        await page.addScriptTag({
```

Comment:
That should no longer be needed now that you use ES modules.

The WebIDL2.js library ends up in `browser.js`.

I note that this may not be such a fantastic idea after all. We'll have to remember to re-build `browser.js` whenever we bump the version of the WebIDL.js library. An alternative would be to define WebIDL2.js as a module here and tell Rollup to exclude it from the build. Anyway, I suggest to just proceed for now (dropping these lines).

But well, I suggest to just proceed for now.